zproject/backends.py

```diff
@@ -1248,6 +1254,7 @@ def social_auth_finish(backend: Any,
     full_name_validated = backend.full_name_validated
     email_address = return_data['validated_email']
     full_name = return_data['full_name']
+    user_avatar_url = return_data.get('user_avatar_url')
```

Comment:
Mypy should be catching, but isn't for some reason, I'm kind of confused by that. this should be .get('user_avatar_url', '') - we don't want this to possibly be `None`

Reply:
Ohhh, I think it's because `return_data` get assigned type `Any` and that makes `return_data.get('user_avatar_url')` get type `Any` as well? Do you want to  try looking this and seeing if we can fix this perhaps with annotating return_data somehow? As a follow-up commit if you find a way to make this work :thinking: